std/format/read.d

```diff
@@ -39,7 +73,11 @@ module std.format.read;
     assert(str.unformatValue!(typeof(null))(spec) == null);
```

Comment:
This is useful if you are, for example, reading code from a file and want to interpret it.

Reply:
Instead of `"%s"` the format string could just be `"null"`. Would work equally well, won't it?

Reply:
I guess, but then you would have to add `null` as a format specifier which really doesn't worth it.
Anyway, this is just a minor case. I don't have any strong opinions.